core/commands/object/object.go

```diff
@@ -326,10 +326,10 @@ and then run
 	},
 
 	Arguments: []cmds.Argument{
-		cmds.FileArg("data", true, false, "Data to be stored as a DAG object").EnableStdin(),
+		cmds.FileArg("data", true, false, "Data to be stored as a DAG object.").EnableStdin(),
 	},
 	Options: []cmds.Option{
-		cmds.StringOption("inputenc", "Encoding type of input data, either \"protobuf\" or \"json\""),
+		cmds.StringOption("inputenc", "Encoding type of input data, either \"protobuf\" or \"Json\"."),
```

Comment:
this one should stay lowercase. Its the actual value we're expecting to be typed in